src/uct/cuda/base/cuda_md.c

```diff
@@ -20,7 +20,7 @@ ucs_status_t uct_cuda_base_detect_memory_type(uct_md_h md, const void *addr,
                                               size_t length,
                                               ucs_memory_type_t *mem_type_p)
 {
-    CUmemorytype memType = 0;
+    CUmemorytype memType = (CUmemorytype)0;
```

Comment:
minor: can set it to `CU_MEMORYTYPE_HOST` by default

Reply:
yes, but we have add static assert for this case